variables.tf

```diff
@@ -74,12 +74,6 @@ variable "enable_eni_prefix" {
   default     = true
 }
 
-variable "annotate_pod_ip" {
-  description = "(Optional) enable to fix pod startup connectivity issue on installing Calico with aws-vpc-cni plugin. (Issue: https://github.com/aws/amazon-vpc-cni-k8s/issues/493)"
-  type        = bool
-  default     = false
-}
-
```

Comment:
Note: Calico Network Policy is replaced by AWS network policy agent, so there is no need to annotate pod IP anymore